frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@apollo/client": "^3.2.1",
     "@monaco-editor/react": "^3.6.2",
+    "@syncfusion/ej2-react-buttons": "^18.2.58",
```

Comment:
This uses a [comercial license](https://github.com/syncfusion/ej2-react-ui-components/blob/master/license). There is a free community license available but it seems like we don't qualify for it. I think we need to use something else.

Reply:
Removed this dependency.